package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "exchange-ftp-authorizer",
   "version": "1.0.0",
-  "license": "GPL-3.0-only",
+  "license": "AGPL-3.0",
```

Comment:
This does match the LICENSE file now.  Though I thought we generally made these smaller library repos MIT?

Reply:
I thought we used MIT for libraries, and AGPL for stand-alone apps. This is a small app, but I think still qualifies as stand alone.